### ISSUE TITLE
At 200% zoom data present in the "Custom accessible properties form" dialog is truncating and not visible: A11y_.NET CoreWinforms_Custom accessible properties form_Zoom

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#11458]
### .NET version

dotnet-sdk-9.0.100-preview.5.24258.1

### Did it work in .NET Framework?

Yes

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_.NET Core[Winforms]_Win32_May2024; #.NETCore; #WCAG1.4.4; #DesktopApp; #Win11; #FTP; #Zoom;

### Environment Details:
App name: .NET Core Winforms
Window Version: Win 11
OS Build: 22621.3085

### Prerequisites:
Display resolution: 1280*768
Scale-> Custom scaling 200%

### Repro Steps:

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "Accessible properties: Testings controls with custom accessible name, role and description" and hit ENTER key.
4. "Custom accessible properties form" screen would appear.
5. Do the resize resolution and observe that At 200% zoom data present in the "Custom accessible properties form" dialog is truncating and not visible.

### Actual Result:
At 200% zoom data present in the "Custom accessible properties form" dialog is truncating and not visible

### Expected Result:
At 200% zoom data present in the "Custom accessible properties form" dialog should be clearly visible.

### User Impact:
Low vision users will not be able to see the controls clearly and will not be getting information related to those.

### Attachment
![11458_A11y_ NET CoreWinforms_Custom accessible properties form_Zoom](https://github.com/dotnet/winforms/assets/90901519/1779ec02-b1b8-43e3-b092-f3b2aa07e238)

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by Olina-Zhang** [comment]
@PoojaNamde The test app and SDK have been updated, please download the updated app and .NET 9.0 SDK to test. It should be as following:
![image](https://github.com/dotnet/winforms/assets/26474449/38e87310-d11e-4bd1-bc3d-347ca858caf7)

**Comment by PoojaNamde** [comment]
@Olina-Zhang Issue should be fixed for 1280 * 768 recommended display resolution as most of the users will be using this resolution. So could you fix it for and assign back to me.

**Comment by Olina-Zhang** [comment]
we tested it in 4k monitor for 200%DPI setting, it is recommended by Windows, as following:
![image](https://github.com/dotnet/winforms/assets/26474449/5afaf9e4-8b93-4c98-8442-3badf0883137)
If display resolution is 1280 * 768 for 200%DPI, there will be an issue with the form not being fully displayed. @merriemcgaw what's your comment for it? Do we want to consider this situation？

**Comment by merriemcgaw** [comment]
@Olina-Zhang can you split this one into a couple of pages that fit into the size requirements? It's a bit of an extra step but better to do it than have this bug appear annually 😄

**Comment by Olina-Zhang** [comment]
@merriemcgaw because of GH issue: https://github.com/dotnet/winforms/issues/11444, we have decided to exclude this form in Accessibility test pass, so do we still need to redesign this form?

**Comment by Tanya-Solyanik** [comment]
@Olina-Zhang, no, we don't need to redesign it, we should remove it completely from this application to avoid further confusion.

**Comment by Olina-Zhang** [comment]
@PoojaNamde finally we decided to remove this "Custom accessible properties form", I have updated and uploaded app. Please verify and close this bug.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:06-13-24;

As per the above comment and text provided for sample form "excluded this Form in Accessibility Test Pass", closing this bug.
